src/main/java/backend/resource/MultiModel.java

```diff
@@ -84,6 +87,11 @@ public synchronized MultiModel replace(List<Model> newModels) {
         return this;
     }
 
+    public synchronized MultiModel replace(Model newModel) {
+        this.add(newModel);
+        return this;
```

Comment:
Yup, `add` is a bit ambiguous in the code context where I used it, provided what it actually does, so I created this method since I don't want to break codes in other places that use `add`.